src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -1147,102 +926,63 @@ internal static SqlConnectionIPAddressPreference ConvertToIPAddressPreference(st
             }
         }
         #endregion
-
-        internal static bool IsValidCertificateValue(string value)
```

Comment:
This method is used in SqlConnectionString in a `#if ADONET_CERT_AUTH` conditional block.